rllib/models/tf/tf_action_dist.py

```diff
@@ -324,6 +325,51 @@ def _unsquash(self, values):
         return unsquashed
 
 
+class Beta(TFActionDistribution):
```

Comment:
Hm what do we use the beta distribution for?

Reply:
As an alternative for SquashedGaussian in PPO or SAC. For double-bounded action spaces.

Reply:
ok, seems like it should be added in a separate PR

Reply:
Done.